graph-platform/neo4j-desktop/neo4j-desktop.adoc

```diff
@@ -44,21 +44,28 @@ Step 2 on the download page instructions shows you how to set up your first proj
 [#desktop-activate]
 == Activation
 
-image::{img}desktop_registration_dec2019.jpg[link="{img}desktop_registration_dec2019.jpg",float="right"]
+[#desktop-first-time]
+== Opening Neo4j Desktop for the First Time
 
 If you are opening Neo4j Desktop for the first time, it should ask you to register the software with an activation key.
-When you downloaded Neo4j Desktop, your unique activation key was displayed at the top of the instructions page.
+This activation key is generated when you first download Neo4j Desktop and will be displayed on the download confimration page.
 While we can copy it to the clipboard and paste it into the provided input in Neo4j Desktop, we also recommend copying it to a safe place as a backup (local file, password vault, or another method).
 
+image::{img}desktop_registration_may2020.jpg[link="{img}desktop_registration_may2020.jpg",float="right"]
+
+Copy this activation key into the Software regisration form and click Activate to continue.
+
+As a new user to Neo4j Desktop, you will be greated with a new sample database containing https://sandbox.neo4j.com/?usecase=recommendations[Movies and Actors].
```

Comment:
not sure if the sandbox link here is rather distracting, perhaps rather link to 

https://github.com/neo4j-graph-examples/movies